recipes/cellpy/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "cellpy" %}
+{% set version = "0.4.0a4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 722ebe3dc25e82b74b9a7538418d3ac32a31658d1544e4110b7058315c7e82de
+
+build:
+  number: 0
+  entry_points:
+    - cellpy=cellpy.cli:cli
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip >=20.0.2
+    - setuptools >=38.6.0
+  run:
+    - python
+    - numpy >=1.15
+    - xlrd >=1.2.0
+    - matplotlib >=3.1.3
+    - cryptography >=2.8
+    - click >=7.1.1
+    - scipy >=1.4.1
+    - pandas >=1.0
+    - setuptools >=38.6.0
+    - ruamel.yaml >=0.15.87
+    - pyodbc >=4.0.16  # [win]
```

Comment:
Looks like this package could be `noarch: python` if not for this selector. We have conda-forge packages for pyodbc on win/mac/linux, any reason not to just depend on this always?

Reply:
Never used pyodbc other than on win (it is used in cellpy for reading MS Access files), but there are no real good reasons for not depending on it in mac/linux also. Good advise. I will remove noarch: python and remove the selector.

Reply:
Sorry, meaning "add noarch: python".